### DISCUSSION TITLE
does it work with GraphQL?

### DISCUSSION THREAD

**Discussion by violabg** [ComfortablyCoding/strapi-plugin-slugify#3] [Q&A] [answered]
cool plug-in, does it work with GraphQL?

**Comment by ComfortablyCoding** [comment]
Hello,

Currently this plugin does not work with GraphQL, but I am looking into adding support for it. Plan to have it supported in v1.1.0

**Comment by ComfortablyCoding** [comment, marked as answer]
v2.0.0 which supports graphql has been released. You can view the full changelog [here](https://github.com/ComfortablyCoding/strapi-plugin-slugify/releases/tag/2.0.0)
